queries/elixir/highlights.scm

```diff
@@ -11,144 +16,184 @@
   "}"
 ] @punctuation.bracket
 
+; Operators
 [
-  ","
-  "->"
+  "!"
+  "!="
+  "!=="
+  "&"
+  "&&"
+  "&&&"
+  "*"
+  "**"
+  "+"
+  "++"
+  "+++"
+  "-"
+  "--"
+  "---"
   "."
-] @punctuation.delimiter
-
-[
-  (sigil_start)
-  (sigil_end)
-  (heredoc_start)
-  (heredoc_end)
-] @punctuation.special
-
-(interpolation ["#{" "}"] @punctuation.special)
-
+  ".."
+  "/"
+  "//"
+  "::"
+  "<"
+  "<-"
+  "<<<"
+  "<<~"
+  "<="
+  "<>"
+  "<|>"
+  "<~"
+  "<~>"
+  "="
+  "=="
+  "==="
+  "=>"
+  "=~"
+  ">"
+  ">="
+  ">>>"
+  "@"
+  "^"
+  "and"
+  "in"
+  "not"
+  "or"
+  "when"
+  "|"
+  "|>"
+  "||"
+  "|||"
+  "~>"
+  "~>>"
+  "~~~"
+  "~"
+] @operator
```

Comment:
Any reason for listing them explicitly instead of matching on relevant nodes? Like [this](https://github.com/elixir-lang/tree-sitter-elixir/blob/7ae20df181b86c79d826abd5aec7a3e32e3d8438/queries/highlights.scm#L34-L51). Either way works fine, though using the nodes should be easier from maintenance perspective.